tests/thin-client/modin-steel-thread.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from snowflake.snowpark import Session
+
+session = Session.builder.create()
+df = pd.DataFrame(
+    [[1, 2, 3, 4, 5, 6, 7], [8, 9, 10, 11, 12, 13, 14], [15, 16, 17, 18, 19, 20, 21]],
+    columns=["A", "B", "C", "D", "E", "F", "G"],
+)
+result = df.iloc[2, 2]
+print(result)
```

Comment:
This test fails right now - I am not sure how to display `result`. It does not produce any other output.

Reply:
Does it work with the main branch? I assume you're running without Phase 1 enabled?